pkg/gpu/aggregator.go

```diff
@@ -0,0 +1,139 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package gpu
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/gpu/model"
+)
+
+const nsecPerSec = 1e9
+
+// aggregator is responsible for receiving stream-level data for one process and aggregating it into metrics
```

Comment:
nit: please add one more line to explicitly mention that process could have multiple streams